raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -889,11 +879,16 @@ private FetchQuorumRecordsResponseData buildEmptyFetchQuorumRecordsResponse(
                 // and we will complete the fetch successfully. Otherwise, if there was
                 // any other error, we need to return it.
                 Errors error = Errors.forException(cause);
+
+                logger.error("Failed to handle fetch from {} at {}", request.replicaId(), request.fetchOffset());
+
                 if (error != Errors.REQUEST_TIMED_OUT) {
                     return buildEmptyFetchQuorumRecordsResponse(error, Optional.empty());
                 }
             }
 
+            logger.error("Completing delayed fetch from {} at {}", request.replicaId(), request.fetchOffset());
```

Comment:
I'm guessing this was left in from debugging? If it's useful, maybe lower level to TRACE?